src/Microsoft.PowerShell.PSReadLine/Words.cs

```diff
@@ -143,24 +131,10 @@ private int FindNextWordPoint(string wordDelimiters)
             if (InWord(i, wordDelimiters))
             {
                 // Scan to end of current word region
-                while (i < _singleton._buffer.Length)
-                {
-                    if (!InWord(i, wordDelimiters))
-                    {
-                        break;
-                    }
-                    i += 1;
-                }
+                for (; i < _singleton._buffer.Length && InWord(i, wordDelimiters); i++) ;
             }
 
-            while (i < _singleton._buffer.Length)
-            {
-                if (InWord(i, wordDelimiters))
-                {
-                    break;
-                }
-                i += 1;
-            }
+            for (; i < _singleton._buffer.Length && !InWord(i, wordDelimiters); i++) ;
```

Comment:
I feel this is easier to read (with the additional parentheses):
for (; (i < _singleton._buffer.Length) && !InWord(I, wordDelimiters); I++);

Reply:
Submit a PR to https://github.com/lzybkr/PSReadLine for any changes in the PSReadLine directory.

Reply:
And I'll likely reject the change - In general I'm not a fan of empty loop bodies.

Reply:
I reverted the file, this change is a anyway just a preference. Maybe I'll try to push it to the right repository. thanks for the link.